drop-transfer/src/ws/server/v5.rs

```diff
@@ -87,7 +87,10 @@ impl<'a> handler::HandlerInit for HandlerInit<'a> {
             .await
             .context("Failed to receive transfer request")?;
 
-        let msg = msg.to_str().ok().context("Expected JOSN message")?;
+        // print msg as ascii
+        debug!(self.logger, "************** msg:\n\t{msg:?}");
```

Comment:
```suggestion
        trace!(self.logger, "************** msg:\n\t{msg:?}");
```

Reply:
I am planning on removing these in the final MR, just desperate to get it working at the moment

Reply:
I am planning on removing these in the final MR, just desperate to get it working at the moment